wordpress-mobile/AztecEditor-iOS
# Fix parsing of poster attribute on videos.

Fixes #964 

To test:
 - Open the regular demo content
 - Check if the video poster images are displayed correctly ( there is two video on the sample content)
